.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        ruby-version: [2.5, 2.6, 2.7, 3.0]
+        ruby-version: [2.6, 2.7, '3.0', 3.1]
```

Comment:
question (non-blocking): Why do we need the quotes around 3.0?

Reply:
not sure 🤔 probably unnecessary
this file is from the upstream repo and doesn't contain any changes from me